### PR TITLE
kumactl 0.5.0

### PR DESCRIPTION
**Pull request by tharun208**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/kumactl.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class Kumactl < Formula
   desc "Kuma control plane command-line utility"
   homepage "https://kuma.io/"
-  url "https://github.com/Kong/kuma.git",
-      :tag      => "0.4.0",
-      :revision => "a37aaadf84e97fa9e5135c31353b51ba6a2245f3"
+  url "https://github.com/Kong/kuma/archive/0.5.0.tar.gz"
```
Comment (chenrui333) [inline Formula/kumactl.rb:4]: Nice!

**Comment by chenrui333** [comment]
@tharun208, can you update the commit message format as `kumactl 0.5.0`?

**Comment by tharun208** [comment]
> @tharun208, can you update the commit message format as `kumactl 0.5.0`?

done

**Review by chenrui333 — APPROVED** [review]

**Comment by tharun208** [comment]
@chenrui333, when I try to install from brew, it is throwing me SHA256 mismatch for one of the bottles

**Comment by SMillerDev** [comment]
Did you change your branch back to master?

**Comment by tharun208** [comment]
> Did you change your branch back to master?

@SMillerDev Thanks, it worked, will keep a note on this

**Pull request closed without merging**